domain/src/main/java/org/fao/geonet/domain/Group.java

```diff
@@ -121,7 +121,7 @@ public Group setId(int id) {
      *
      * @return group name
      */
-    @Column(nullable = false, length = 32)
+    @Column(nullable = false, length = 255)
```

Comment:
you mentioned 256 in the README here: https://github.com/georchestra/georchestra/commit/8a4f30de5a88af765f1aa2baa099836e730fa01d

Reply:
Right, fixed with b3d57edc5f528b972c50e73e4b66467763bed3fa